data/fetch_recipe.py

```diff
@@ -0,0 +1,91 @@
+import requests
+import json
+import re
+import sys
+from src.item_db import item_db
+from src.log_setup import logger
+
+logger.debug("Running fetch_recipe.py...")
+
+API = "https://terraria.wiki.gg//api.php"
+PARAMS = {
+    "action": "cargoquery",
+    "format": "json",
+    "tables": "Recipes",
+    "fields": "result, resultid, station, args",
+    "offset": 0,
+}
+
+S = requests.Session()
+R = S.get(url=API, params=PARAMS)
+DATA = R.json()
```

Comment:
I'd change the S and R from capital case to small case. We usually use capital to variables that store constants, and S,get is an operation you are redoing later on.